.github/workflows/release.yml

```diff
@@ -284,9 +284,8 @@ jobs:
     - name: Srtool build
       id: srtool_build
       uses: chevdor/srtool-actions@v0.7.0
-      env:
-        BUILD_OPTS: -Z unstable-options
       with:
+        tag: 1.66.1
```

Comment:
Can we please add a TODO here to remove this once we uplift to latest polkadot?

Reply:
added by @shunsukew